proto/message_api/v1/message_api.proto

```diff
@@ -7,139 +7,139 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 
 option go_package = "github.com/xmtp/proto/v3/go/message_api/v1";
 option java_package = "org.xmtp.proto.message.api.v1";
-
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
-    info: {
-        title: "MessageApi";
-        version: "1.0";
-    };
+  info: {
```

Comment:
Not sure why my formatter is different from what was used for this file. We should probably add some vscode config to standardize everyone